README.md

```diff
@@ -13,13 +13,13 @@ Contributing and Communications
 * The [Issues and Milestones](https://github.com/meteor-gazelle/meteor-gazelle/issues) contains items that need to be worked on. Check there for a general overview of what needs to be worked on.
 * IRC
   * &#35;gazelle on irc.what-network.net is the official meteor-gazelle development channel.
-  * If you are looking for something to work on, ping a Core team member (operators in #gazelle).
+  * If you are looking for something to work on, ping the core team using !core <message>, or create a github issue labeled appropriately.
```

Comment:
Github issues wouldn't be created by someone when they are looking for work. The milestones and existing issues indicate what there is to be worked on. Issues can be created for questions, discussions, etc (labeled correctly) which would otherwise have taken place in irc.